app/scripts/modules/core/src/presentation/forms/layouts/ResponsiveFieldLayout.tsx

```diff
@@ -27,8 +27,9 @@ export function ResponsiveFieldLayout(props: ILayoutProps) {
         </div>
         <div className="sp-formItem__right">
           <div className="sp-form">
-            <span className="field">
-              {input} {actions}
+            <span className="flex-container-h margin-between-lg field">
```

Comment:
I don't think this is quite right for mobile, but I also think it was broken before, so /shrug

Reply:
To my untrained eye it looks...at least not worse than before?

![e2ME35kYs25](https://user-images.githubusercontent.com/15936279/71919040-aba43980-3151-11ea-8cda-9b2ac9419622.png)